README.md

```diff
@@ -1,21 +1,68 @@
-# DevOps Apprenticeship: Project Exercise
+# DevOps Apprenticeship: Running Jason B's ToDo App
 
-## Getting started
+## System Requirements
 
-The project uses a virtual environment to isolate package dependencies. To create the virtual environment and install required packages, run the following from a bash shell terminal:
+The project uses poetry for Python to create an isolated environment and manage package dependencies. To prepare your system, ensure you have an official distribution of Python version 3.7+ and install poetry using one of the following commands (as instructed by the [poetry documentation](https://python-poetry.org/docs/#system-requirements)):
+
+### Poetry installation (Bash)
+
+```bash
+curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+```
+
+### Poetry installation (PowerShell)
+
+```powershell
+(Invoke-WebRequest -Uri https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py -UseBasicParsing).Content | python
+```
+
+## Dependencies
+
+The project uses a virtual environment to isolate package dependencies. To create the virtual environment and install required packages, run the following from your preferred shell:
+
+```bash
+$ poetry install
+```
+
+You'll also need to clone a new `.env` file from the `.env.tempalate` to store local configuration options. This is a one-time operation on first setup:
 
-### On macOS and Linux
 ```bash
-$ source setup.sh
+$ cp .env.template .env  # (first time only)
 ```
-### On Windows (Using Git Bash)
+
+The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
+
+
+Check for additional dependencies
+
+Packages required:
+```bash
+  * Requests
+  * Pytest
+  * Dotenv
+  * Selenium
+```
```

Comment:
You don't need to include this here. There is no action for the reader - `poetry install` will install them all - and there is a more reliable source of truth for this: pyproject.toml